### PR TITLE
feat: add command line parser

### PR DESCRIPTION
**Pull request by benjamin-craig**

### CODE REVIEW

File: stix_shifter.py [hunk 1]
```diff
@@ -1,48 +1,80 @@
...
+        result = self.translate(self.args.module, self.args.action, self.args.data)
+        print(result)
+        exit(0)
 
-    interface = translator_module.Translator()
+    def translate(self, module, translation_type, data):
+        """
+        Translated queries to a specified format
+        :param module: What module to use
+        :type module: one of MODULES 'qradar', 'dummy'
```
Comment (Logicon211) [inline stix_shifter.py]: Should also include aql_passthrough here as well

Comment (benjamin-craig) [reply stix_shifter.py]: I added that now.

**Comment by benjamin-craig** [comment]
@jleemur we need aql_passthrough in the case we need to do an aql query and not translate it to stix.  The aql pass trough will still translate results to Stixs.  we also need the dummy module as an example.

File: stix_shifter.py [hunk 2]
```diff
@@ -1,48 +1,80 @@
-# Main module to be called
 import sys
 import importlib
+import argparse
 
-MODULES = ['qradar', 'dummy']
+MODULES = ['qradar', 'dummy', 'aql_passthrough']
```
[the remark this answers is not in the document]
Comment (benjamin-craig) [reply stix_shifter.py]: Yea I was just getting ready for Logan's PR.

File: README.md [hunk 3]
```diff
@@ -8,19 +8,32 @@ Requires Python 3.6
...
+
+positional arguments:
+  {qradar,dummy,aql_passthrough}
+                        What translation module to use
+  {results,query}       What translation action to perform
+  data                  The data to be translated
+
+optional arguments:
+  -h, --help            show this help message and exit
+```
```
[the remark this answers is not in the document]
Comment (benjamin-craig) [reply README.md]: will do.

**Pull request merged**